package/gluon-wifi-encrypt/Readme.md

```diff
@@ -0,0 +1,15 @@
+# Gluon WiFi encrypt
+Encrypt wireless networks in a non-Freifunk setup.
+
+This package allows to encrypt 802.11s and infrastructure networks using a pre shared key (psk).
+
+This setup results in passwords being stored in plain-text. Site-configuration, Firmware-downloads and
+Devices must be protected against unauthorized access
+
+*make sure to use exclude hostapd-mini site.mk*
+
+Example configuration:
+
+```lua
+encrypt_wifi = { mesh = '802.11s.passwd', ap = ' infrastructure.passwd', },
```

Comment:
`wlan_encryption = { mesh = { mode = 'psk', key = 'mesh-example-key' }, client = {  mode = 'psk', key = 'client-example-key' }}`

Reply:
Maybe better put this in the existing WLAN site.conf sections.